src/System.Numerics.Dec64/System.Numerics.Dec64.csproj

```diff
@@ -1,10 +0,0 @@
-<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Are we removing Dec64 for good or just moving it to some temporary location? If we are removing it, I think we need to contact the owner first.

Reply:
There is no steward within Microsoft for this.

@benaadams initiated it but it has been inactive since then:
https://github.com/dotnet/corefxlab/pull/764
https://github.com/dotnet/corefx/issues/10951